build.gradle.kts

```diff
@@ -1,7 +1,8 @@
 buildscript {
   repositories {
     google()
-    jcenter()
+    jcenter() // Remove, see mapbox-maps-android#92
```

Comment:
Can we remove it already?

Reply:
nope, some upstream dependencies are still not hosted yet on mavencentral..